docs/querying-data.md

```diff
@@ -0,0 +1,66 @@
+# Querying Data
+Once you've identified the data you want, you'll want to start retrieving it and putting it to use. The following methods should help you get the data in the format that you want.
+
+## Get data points
+`get_data_points(**selection)` is the primary method for retrieving data. The [code snippets](./searching-data#code-snippets) feature covered earlier provides you with a fully completed `get_data_points()` query, such as:
+```py
+# Wheat - Area Harvested (area) - India (USDA PS&D)
+client.get_data_points(**{
+    'metric_id': 570001,
+    'item_id': 95,
+    'region_id': 1094, 
+    'source_id': 14, 
+    'frequency_id': 9
+})
+```
+The above query has completed fields for `metric_id`, `item_id`, `region_id`, `source_id`, and `frequency_id`. However, `get_data_points()` can also accept fields to further narrow your data series of interest: `partner_region_id` (used only in series that represent a flow between two places), `start_date`, `end_date`, [`show_revisions`](#show-revisions), [`insert_null`](https://gro-intelligence.github.io/api-client/development/api.html#api.client.lib.get_data_points), and [`at_time`](https://gro-intelligence.github.io/api-client/development/api.html#api.client.lib.get_data_points).

```

Comment:
Note: after I merge https://github.com/gro-intelligence/api-client/pull/121 the links to these functions on the API reference page will change. Let's come back to that as a follow-up

Reply:
Updated with new links